capacity4more/modules/c4m/restful/c4m_restful_quick_post/components/c4m-app/src/controllers/main.js

```diff
@@ -162,6 +162,23 @@ angular.module('c4mApp')
       QuickPostService.togglePopover(name, event, $scope.popups);
     };
 
+    /**
+     * Check if current category has at least ont selected child.
+     *
+     * @param key
+     *  Category term id.
+     *
+     * @returns {boolean}
+     */
+    $scope.categoryHasChildrenSelected = function(key) {
+      var result = false;
+      angular.forEach($scope.categories[key].children, function(child, childKey) {
```

Comment:
maybe it's better to have a ``for`` loop here, so you could return early as you find the first ``true``